reana_commons/models.py

```diff
@@ -213,18 +213,18 @@ class Run(Base, Timestamp):
                  default=generate_uuid)
     workflow_uuid = Column(UUIDType, primary_key=True)
     run_number = Column(Integer)
-    current_command = Column(String(1024))
-    current_command_idx = Column(Integer)
-    current_job = Column(UUIDType)
-    total_commands = Column(Integer)
-    current_step = Column(Integer)
-    total_steps = Column(Integer)
+    planned = Column(Integer)
```

Comment:
`planned` may be understood in several ways... is it `total = planned + submitted` and `submitted = succeeded + failed + currently-running`?

Reply:
So the progress is then, roughly: 

- total = submitted + failed + succeeded + planned
- current step: failed + succeeded + submitted /  total